### ISSUE TITLE
proposal: add `eslint-plugin-import`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/eslint-config-stylelint#241]
### What is the problem you're trying to solve?

1. ordering `import`s, `export`s last, etc.
2. establishing guidelines
3. not having to enforce them—anymore—during code reviews

### What solution would you like to see?

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unused-modules.md

after v16
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-commonjs.md

**Comment by ybiquitous** [comment]
Thanks for opening the issue. Sounds good to me. 👍🏼

**Comment by Mouvedia** [comment]
@ybiquitous stylelint/stylelint#6942
Should it remain stylelint specific? If so this issue can be closed.

**Comment by ybiquitous** [comment]
> Should it remain stylelint specific?

I think we should use the same rule for importing within the organization. So, this proposal makes sense.

But adding the importing rules to `stylelint` first is a good try. If so, people can realize the value of the new rules.

**Comment by Mouvedia** [comment]
1. stylelint/stylelint: PR
2. after merge, pick the ones relevant for the org
3. PR here
4. stylelint/stylelint: update dependency
5. stylelint/stylelint: removal of the duplicates

Sounds good?

**Comment by ybiquitous** [comment]
Yes, but can you please wait for PRs (rewriting to ESM) into the `v16` branch? Probably, so many conflicts would occur.
